### PR TITLE
unify docs and fix non markdown links

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
unified docs and fixed link to non markdown files

### CODE REVIEW

File: demos/gesture_recognition_demo/python/README.md [hunk 1]
```diff
@@ -82,17 +82,17 @@ To run the demo, you can use public or pre-trained models. To download the pre-t
...
+-m_a <path_to_model>/asl-recognition-0004.xml \
+-m_d <path_to_model>/person-detection-asl-0001.xml \
 -i 0 \
 -c <omz_dir>/data/dataset_classes/msasl100.json
 ```
 
 The demo starts in person tracking mode and to switch it in the action recognition mode you should press `0-9` button with appropriate detection ID (the number in top-left of each bounding box). If frame contains only one person, they will be chosen automatically. After that you can switch back to tracking mode by pressing space button.
 
-An example of file with class names can be found [here](../../../data/dataset_classes/msasl100.json).
+An example of file with class names can be found within OMZ folders tree, <omz_dir>/data/dataset_classes/msasl100.json.
```
Comment (IRDonch) [inline demos/gesture_recognition_demo/python/README.md:93]:
```suggestion
An example of file with class names can be found within the OMZ directory at `<omz_dir>/data/dataset_classes/msasl100.json`.
```

File: demos/gesture_recognition_demo/python/README.md [hunk 2]
```diff
@@ -82,17 +82,17 @@ To run the demo, you can use public or pre-trained models. To download the pre-t
...
 
-> **NOTE**: To run the demo application with video examples of gestures specify the `-s` key with valid path to the directory with video samples (you can find some ASL gesture video samples [here](https://github.com/intel-iot-devkit/sample-videos)). The name of each video sample should be the valid name of gesture from the [`msasl100.json`](../../../data/dataset_classes/msasl100.json) file. To navigate between samples use 'f' and 'b' keys for iterating next and previous respectively video sample.
+> **NOTE**: To run the demo application with video examples of gestures specify the `-s` key with valid path to the directory with video samples (you can find some ASL gesture video samples [here](https://github.com/intel-iot-devkit/sample-videos)). The name of each video sample should be the valid name of gesture from <omz_dir>/data/dataset_classes/msasl100.json file. To navigate between samples use 'f' and 'b' keys for iterating next and previous respectively video sample.
```
Comment (IRDonch) [inline demos/gesture_recognition_demo/python/README.md:95]:
```suggestion
> **NOTE**: To run the demo application with video examples of gestures specify the `-s` key with valid path to the directory with video samples (you can find some ASL gesture video samples [here](https://github.com/intel-iot-devkit/sample-videos)). The name of each video sample should be the valid name of gesture from `<omz_dir>/data/dataset_classes/msasl100.json` file. To navigate between samples use 'f' and 'b' keys for iterating next and previous respectively video sample.
```

File: demos/text_to_speech_demo/python/README.md [hunk 3]
```diff
@@ -55,11 +55,25 @@ Options:
...
+# WaveRNN upsample
 ```
-python3 text_to_speech_demo.py --model_duration forward_tacotron_duration_prediction.xml --model_forward forward_tacotron_regression.xml --input <path_to_file_with_text.txt> wavernn --model_upsample wavernn_upsampler.xml --model_rnn wavernn_rnn.xml
+python3 text_to_speech_demo.py \
+--input <path_to_file>/text.txt \
+-o <path_to_audio>/audio.wav \
+--model_duration <path_to_model>/forward_tacotron_duration_prediction.xml \
+--model_forward <path_to_model>/forward_tacotron_regression.xml \
+--model_upsample <path_to_model>/wavernn_upsampler.xml \
+--model_rnn <path_to_model>/wavernn_rnn.xml
```
Comment (IRDonch) [inline demos/text_to_speech_demo/python/README.md:67]: This should be indented.

File: demos/text_to_speech_demo/python/README.md [hunk 4]
```diff
@@ -55,11 +55,25 @@ Options:
...
 ```
+# MELGAN upsample
 ```
-python3 text_to_speech_demo.py -m_duration forward_tacotron_duration_prediction_att.xml -m_forward forward_tacotron_regression_att.xml -i <path_to_file_with_text.txt> -o <path_to_audio.wav> melgan -m_melgan melganupsample.xml
+python3 text_to_speech_demo.py \
+-i <path_to_file>/text.txt \
+-o <path_to_audio>/audio.wav \
+-m_duration <path_to_model>/forward_tacotron_duration_prediction_att.xml \
+-m_forward <path_to_model>/forward_tacotron_regression_att.xml \
+-m_melgan <path_to_model>/melganupsample.xml
```
Comment (IRDonch) [inline demos/text_to_speech_demo/python/README.md:76]: This should also be indented.

File: demos/text_to_speech_demo/python/README.md [hunk 5]
```diff
@@ -55,11 +55,25 @@ Options:
 Running the application with the empty list of options yields the usage message and an error message.
 
 ## Example for running with arguments
+
+# WaveRNN upsample
```
Comment (IRDonch) [inline demos/text_to_speech_demo/python/README.md:59]:
```suggestion
### ForwardTacotron with WaveRNN
```

File: demos/text_to_speech_demo/python/README.md [hunk 6]
```diff
@@ -55,11 +55,25 @@ Options:
...
-python3 text_to_speech_demo.py --model_duration forward_tacotron_duration_prediction.xml --model_forward forward_tacotron_regression.xml --input <path_to_file_with_text.txt> wavernn --model_upsample wavernn_upsampler.xml --model_rnn wavernn_rnn.xml
+python3 text_to_speech_demo.py \
+--input <path_to_file>/text.txt \
+-o <path_to_audio>/audio.wav \
+--model_duration <path_to_model>/forward_tacotron_duration_prediction.xml \
+--model_forward <path_to_model>/forward_tacotron_regression.xml \
+--model_upsample <path_to_model>/wavernn_upsampler.xml \
+--model_rnn <path_to_model>/wavernn_rnn.xml
 ```
+# MELGAN upsample
```
Comment (IRDonch) [inline demos/text_to_speech_demo/python/README.md:69]:
```suggestion
### Modified ForwardTacotron with MelGAN
```

File: models/public/aclnet-int8/aclnet-int8.md [hunk 7]
```diff
@@ -3,7 +3,7 @@
 ## Use Case and High-Level Description
 
 The `AclNet-int8` model is quantized and fine-tuned with NNCF variant of [AclNet](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/aclnet/aclnet.md) model, which is designed to perform sound classification.
-The `AclNet-int8` model is trained on an internal dataset of environmental sounds for 53 different [classes](../../../data/dataset_classes/aclnet.txt).
+The `AclNet-int8` model is trained on an internal dataset of environmental sounds for 53 different classes, listed in file <omz_dir>/data/dataset_classes/aclnet.txt.
```
Comment (IRDonch) [inline models/public/aclnet-int8/aclnet-int8.md:6]:
```suggestion
The `AclNet-int8` model is trained on an internal dataset of environmental sounds for 53 different classes, listed in file `<omz_dir>/data/dataset_classes/aclnet.txt`.
```

(ditto in several occurrences below)

**Review by IRDonch — CHANGES_REQUESTED** [review]
Thanks for taking care of this; I didn't realize we had that many inconsistencies.

File: demos/text_to_speech_demo/python/README.md (hunk 3 above)
Comment (vladimir-dudnik) [reply demos/text_to_speech_demo/python/README.md]: fixed

File: demos/text_to_speech_demo/python/README.md (hunk 4 above)
Comment (vladimir-dudnik) [reply demos/text_to_speech_demo/python/README.md]: fixed

File: models/public/aclnet-int8/aclnet-int8.md (hunk 7 above)
Comment (IRDonch) [reply models/public/aclnet-int8/aclnet-int8.md]: Update the other occurrences too.

**Review by IRDonch — APPROVED** [review]

**Pull request merged**